cmd/get.go

```diff
@@ -81,14 +84,24 @@ and provides a fast and easy alternative to a package manager.`,
 
 		outFilePath := path.Join(tmp, fileName)
 
-		out, err := os.Create(outFilePath)
-		if err != nil {
-			return err
-		}
-		defer out.Close()
+		if tool.NeedDecompression == true {
```

Comment:
Could this be a method instead? `IsArchive()`? Look at the URL's suffix?

Reply:
Yes, will do. Initially I want to look at the mime type but haven't figure out how to do that with go without third party lib.